copy_test.go

```diff
@@ -47,6 +47,104 @@ func TestStructToStruct_PtrToInt(t *testing.T) {
 	}, dst)
 }
 
+func TestStructToStruct_StructToPointer(t *testing.T) {
+	v15 := 15
+	v42 := 42
+
+	type N struct {
+		Field1 int
+	}
+	type S struct {
+		Field1 N
+		Field2 int
+	}
+	src := &S{
+		Field1: N{
+			Field1: v15,
+		},
+		Field2: v42,
+	}
+	type SN struct {
+		Field1 *int
+	}
+	type D struct {
+		Field1 *SN
+		Field2 *int
+	}
+	dst := new(D)
+
+	mask := fieldmask_utils.MaskFromString("Field1,Field2,Field1.Field1")
```

Comment:
`MaskFromString` has a special syntax (e.g. `Field1{Subfield1, Subfield2}, Field2}`) and is intended to be used in tests only.

Please, update to the following:
```suggestion
	mask := fieldmask_utils.MaskFromString("Field1,Field2")
```